aspnetcore/client-side/yeoman.md

```diff
@@ -74,25 +74,25 @@ Build and run the app using `dotnet build` and `dotnet run`:
 
 ![Command window](yeoman/_static/dotnet-build-run.png)
 
-At this point you can navigate to the URL shown to test the newly created ASP.NET Core app.
+At this point, you can navigate to the URL shown to test the newly-created ASP.NET Core app.
 
 ## Client-side packages
 
-The front end resources are provided by the templates from the yeoman generator using the [Bower](bower.md) client-side package manager, adding *bower.json* and *.bowerrc* files to restore client-side packages using the [Bower](bower.md) client-side package manager.
+The front-end resources are provided by the templates from the Yeoman generator using the [Bower](xref:client-side/bower) client-side package manager, adding *bower.json* and *.bowerrc* files to restore client-side packages using Bower.
 
-The [BundlerMinifier](https://github.com/madskristensen/BundlerMinifier/wiki) component is also included by default for ease of concatenation (bundling) and minification of CSS, JavaScript and HTML.
+The [BundlerMinifier](https://github.com/madskristensen/BundlerMinifier/wiki) component is also included by default for ease of concatenation (bundling) and minification of CSS, JavaScript, and HTML.
```

Comment:
Probably should link to the bundlerminifier doc - xref:client-side/bundling-and-minification.md here.